### ISSUE TITLE
feature: add a "question" template for getting input from user in templates

### ISSUE THREAD

**Issue by davidovich** [davidovich/summon#84]
I see it practical that we provide a "question" template function that could be used to ask user about inputting info

Used in a summon file something like:

```
namespace: {{ question "ns" "what namespace should be used" "default" }}
cluster: {{ question "cluster" "please specify the destination cluster" [splice of choices] }}
```

**Comment by davidovich** [comment]
Questions or `input` can be pre-defined or inlined. Inlining is order dependant, naturally. Pre-defined `inputs` are invoked at program startup time. One would prefer to design the cli with the former as it would be less surprising than in-the-middle execution stops for prompting. Inlining would be used in conditional scenarios where an input is only needed for a certain part of the cli hierarchy, or optional data.

**Comment by davidovich** [comment]
closed by #102

**Comment by davidovich** [comment]
closed by #102
